### DISCUSSION TITLE
CPBR12 - Campus Party / 12 de Fev

### DISCUSSION THREAD

**Discussion by Juuhz** [frontendbr/forum#1334] [Discussão]
Fala Devs, tudo jóia ? 

Estou abrindo este tópico para reunir a galera que irá para Campus Party em fevereiro, assim como eu. Deixo aqui algumas sugestões de discursões para propor a este tópico: 

1) Quero muito participar dos hackathons, porém ainda não tenho equipe formada, alguém topa montar uma equipe ? Se sim, deixe ai suas especialidades. 

2) Quais palestras serão o foco de vocês ? 

3) Já foram ? Se sim, pode passar algumas dicas ? ( Essa é minha primeira CPBR ).

4) Da para comer bem lá ? 

5) Topam marcar um ponto de encontro para fazermos uma resenha ? 

----

OBS: Esses são alguns pontos para discursões, mas caso tenham outros, fiquem a vontade ;) 
OBS2: Se você não sabe oque é, veja: https://brasil.campus-party.org/

Atenciosamente,
Reinaldo Amorim
http://reinaldoamorim.com.br

**Comment by heliomsolivas** [comment]
Eu tenho uma puta vontade de ir, tava pensando em ir 1/2 dias (mas não entendi direito como funciona), alguém que já foi consegue dar um help pra gente? \o/

**Comment by Juuhz** [comment]
Fala @heliomsolivas,

Mesmo sendo minha primeira Campus, eu li muito sobre e posso te passar alguns insights, quais são suas dúvidas ?

Abraços.

**Comment by vNNi** [comment]
Também tenho vontade de ir, mas como o @heliomsolivas, tenho dúvidas.
Tu vai e tem palco pra palestra, depois rola o hackathon enquanto as palestrar rolam? 
Sobre o Hackathon, eles fecham uma tecnologia X e tu tem que fazer em cima dela?

**Comment by heliomsolivas** [comment]
@Juuhz as palestras acontecem o tempo todo? É valido pra quem não é da área de dev?

**Comment by Juuhz** [comment]
@heliomsolivas e @vNNi 

As palestras acontecem o tempo todo, mas elas tem agendas e ocorrem até um certo horário ( acho que a última palestrado do dia ocorre por volta das 22hs ). 

Tirei um print de algumas palestras que irão ocorrer:

![image](https://user-images.githubusercontent.com/7996030/50832740-3fbca480-1336-11e9-9f56-e043351dfee2.png)

Conforme podemos ver e respondendo sua dúvida @heliomsolivas, as palestras não são necessariamente voltadas para área de desenvolvimento, e sim voltada para tecnologia, oque abrange uma gama maior de assuntos. 

OBS: Essa ai é apenas 1 página de palestras, tem mais de 6 páginas dessa com outros assuntos. Para ver todas: https://campuse.ro/events/campus-party-brasil-2019/talk

---

Sobre os hackathon: Não sei ao certo os horários mas acredito que é dado um prazo para a entrega do projeto e o pessoal vai fazendo ao longo do evento. 

@vNNi normalmente é uma empresa que fecha um tema e os grupos tem que fazer soluções inteligentes sobre esse tema. Veja o do ano passado: https://www.visa.com.br/mais-visa/sobre-a-visa/sala-de-imprensa/vencedores-do-hackathon-visa-realizado-na-campus-party.html

Abraços.

**Comment by ninetails** [comment]
Bom, fui ano passado e tirando a primeira, acho que só não fui em mais uma.

Como "veterano" (não em questão de status, mas de fazer tempo que participo), pra mim ficou meio que mais do mesmo, não tenho a mesma empolgação que as primeiras Campus que fui, porém ainda gosto da experiência.

Outro porém é que como mudou o local, não posso dizer que será como nos outros anos, mas vou focar em como foi anteriormente.

----

Complementando aos insights do @Juuhz , sobre palestras: tem uma agenda no site (não lembro se no campus-party.org ou na área de campuseiro logado) onde vc pode ver quais temas e fazer sua própria agenda (ou importar pro Google Calendar, se n me engano). Os temas são diversos, tem YouTubers ( ¯\_(ツ)_/¯ ), tem Keynotes (tá meio fraco ultimamente, mas nos primeiros já estiveram Kevin Mitnick, Buzz Aldrin, Al Gore...) Pros não keynotes, são palestras abertas, em palcos abertos (ou workshops) e geralmente qualquer um pode colar e assistir (dependendo do workshop pode ser que tenha vagas limitadas).

Um problema das palestras serem abertas por opção da organização é que não há isolamento acústico, dependendo do lugar que vc sentar, irá ouvir a palestra do lado ou o palco principal

Os Hackatons geralmente no primeiro dia abre inscrições e no penúltimo dia fazem as premiações... Não cheguei a participar de nenhum, então não sei dar mais detalhes.

----

Tem eventos paralelos sobre empreendedorismo e uma área para Startups se divulgarem.

Tem brindes (estilo adesivos) a rodo e coisas que marqueteiros adoram do tipo engajamento via Twitter e fazer campuseiro ficar correndo.

**Camping, caso tiver pego**

te dão uma barraca que vc pode levar pra casa e recomendo levar um colchão inflável. Tem lugar pra tomar banho mas dependendo do horário que vc chegar (ou por sorte/falta de) pode ser que fique num lugar onde o banheiro e o banho ficam longe da sua barraca.

_edit:_ Bom levar cadeado, [por levar mesmo](https://youtu.be/wpIJVWXsBBI)

**Alimentação**

Eles contratam somente UMA empresa para cuidar dessa parte. Terá alguns quiosques 24h (geralmente casa do pão de queijo e alguma outra) mas são bem caros. A alternativa dada por eles é pagar o catering, que é _equals_ bandejão.

Porém esse ano será no Expo Center Norte, que fica perto de um Shopping com muitas alternativas, e um supermercado (Carrefour).

**Hardware**

Recomendo levar note, porém caso tiver Desktop também não há problemas em levar. Há sim problemas com furtos, então sempre é bom ficar esperto. Estamos no Brasil, _deal with it_. Caso levar note, ao entrar e sair sempre há uma verificação para validar seu equipamento.

Sobre validação de equipamento, quando vc chega é efetuado um cadastro do seu equipamento. São taggeados: Notebook, Desktop, Monitor, Console (dos que eu lembro)

Também há comércio interno entre campuseiros, caso tenha esquecido algo ou precisa de um cadeado pra note.

**DC++**

Sempre sobem um servidor para compartilhamento de arquivos interno e também é usado como chat do evento (além do Twitter).

**Horários**

Durante o evento, pode sair e entrar a qualquer momento. Tem ponto de Taxi na porta (creio que em todos os pavilhões). Uber e 99 sem problemas também. Dependendo alguma destas podem ter inclusive parceria com o evento.

Aconselho também chegar e sair em horários alternativos, pois tanto no checkin quanto no checkout as filas são meio chatas.

**Campus B**

Não pode entrar com bebidas alcoólicas no evento, então todas as noites sempre tem uns vendedores de rua que ficam na porta do evento. Não só bebidas alcoólicas como dogão/hamburgueres, pizza (num valor bem abaixo do Trusteado) etc. E a galera que não se isola nos jogos/assistindo o que baixou se reune pra fazer um social.

----

Tentei fazer um apanhado geral + guia pra quem for de primeira viagem.

Topo encontrão sim :D

Se tiverem mais dúvidas, estamos aí.

**Comment by Juuhz** [comment]
@ninetails,

Muito obrigado pelos insights, foram absurdamente fodas \o/, eu não sabia que tinha que ir no site e confirmar a participação nas palestras, fui correndo para montar minha agenda, seguindo sua dica, obrigado <3

Se tu for esse ano, bora participar de algum hackathon, eu irei com um amigo que é DEV também.

**Comment by ninetails** [comment]
@Juuhz opa precisa confirmar não, é só colar rs... só sugeri fazer sua agenda pra ficar certinho no Google Calendar e vc se programar. Vc também pode assistir da própria bancada (pois a maioria faz stream via YouTube)

Só o Hackaton que n dou certeza de participar auehuaehuae geralmente vou pra ficar de boas

**Comment by NicholasKuchiniski** [comment]
Provavelmente vou. Sabem me dizer como funciona o hackaton ?

**Comment by rsmelo92** [comment]
@NicholasKuchiniski Não sei como funciona na CPBR mas na Campus Party local que aconteceu em salvador Bahia, os Hackathons foram organizados por empresas, uma foi do Cappra Institute (a que paticipei e meu time conseguiu o primeiro lugar) e outras duas do governo do estado da Bahia. As inscrições são próximas do evento e é só se inscrever, aparecer lá no dia e local combinado, ouvir as instruções e entregar alguma coisa até a data estipulada. Recomendo, é uma experiência muito boa

**Comment by NicholasKuchiniski** [comment]
> As inscrições são próximas do evento e é só se inscrever, aparecer lá no dia e local combinado, ouvir as instruções e entregar alguma coisa até a data estipulada.

@rsmelo92 As inscrições são feitas por aquele portal deles na área challenge ? (campuse.ro) vou ficar atento lá para me inscrever. Espero que seja desse jeito que você falou mesmo, vou adorar a experiência com certeza... Único problema é que sou menor de idade emancipado e tenho dúvidas se isso pode dar problemas na hora de se inscrever (visto que observei alguns regulamentos e pediam 18 anos ou mais, não existia nem uma parte falando de exceção). Vou fazer como o @Juuhz  indicou e enviar um email para eles perguntando.

**Comment by rsmelo92** [comment]
@NicholasKuchiniski irmãozinho abriram as inscrições https://brasil.campus-party.org/hackathon/

**Comment by NicholasKuchiniski** [comment]
> @NicholasKuchiniski irmãozinho abriram as inscrições https://brasil.campus-party.org/hackathon/

@rsmelo92 Opa, valeu!

**Comment by Juuhz** [comment]
Boa @rsmelo92! 

Me inscrevi já \o/

----

@NicholasKuchiniski, conseguiu tirar sua dúvida com a galera da campus ? vai poder participar dos hackathons ?

**Comment by NicholasKuchiniski** [comment]
> @NicholasKuchiniski, conseguiu tirar sua dúvida com a galera da campus ? vai poder participar dos hackathons ?

@Juuhz Infelizmente os regulamentos não permitem menores de 18. Mas consigo entrar no evento.
Acho que vou deixar pro ano que vem que já vou estar fazendo faculdade e com conhecimentos mais consolidados, mas boa sorte para quem for participar, ví o link que o @rsmelo92 mandou e ja tem um data science ali no meio, vou acompanhar, vai sair coisa muito massa tenho certeza.
